src/graphics/esthetic.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef ESTHETIC_HPP
+#define ESTHETIC_HPP
+
+namespace graphic {
+
+enum Coordinate { x, y };
+
+inline constexpr int GLOBAL_HEIGHT  = 600;
+inline constexpr int GLOBAL_WIDTH   = 800;
+inline constexpr int PAD_SHIFT      = 6;
+inline constexpr int BASE_SHIFT     = 10;
+inline constexpr int AXIS_THICKNESS = 2;
+inline constexpr int AXIS_SHIFT     = 20;
+inline constexpr int AXIS_SPACE     = 20;
```

Comment:
For `constexpr` variables the `static` keyword is better (guarantees that the value will be evaluated at compile-time)

Reply:
changed